Resources/Prototypes/Catalog/Fills/Backpacks/StarterGear/backpack.yml

```diff
@@ -1,313 +1,323 @@
-
-# ERT
-
-- type: entity
-  noSpawn: true
-  parent: ClothingBackpackERTLeader
-  id: ClothingBackpackERTLeaderFilled
-  components:
-    - type: StorageFill
-      contents:
-        - id: BoxSurvival
-        - id: WeaponPulseCarbine
-        - id: WeaponPulsePistol
-        - id: WeaponDisabler
-        - id: MedicatedSuture
-        - id: RegenerativeMesh
-        - id: Handcuffs
-        - id: CrowbarRed
-
-- type: entity
-  noSpawn: true
-  parent: ClothingBackpackERTSecurity
-  id: ClothingBackpackERTSecurityFilled
-  components:
-    - type: StorageFill
-      contents:
-        - id: BoxSurvival
-        - id: WeaponDisabler
-        - id: WeaponPulsePistol
-        - id: WeaponPulseCarbine
-        - id: MedicatedSuture
-        - id: RegenerativeMesh
-        - id: BoxHandcuff
-        - id: CrowbarRed
-
-- type: entity
-  noSpawn: true
-  parent: ClothingBackpackERTMedical
-  id: ClothingBackpackERTMedicalFilled
-  components:
-    - type: StorageFill
-      contents:
-        - id: BoxSurvivalMedical
-        - id: WeaponPulseCarbine
-        - id: Hypospray
-        - id: MedkitAdvancedFilled
-        - id: CrowbarRed
-        - id: OmnizineChemistryBottle
-        - id: EpinephrineChemistryBottle
-        - id: EpinephrineChemistryBottle
-
-- type: entity
-  noSpawn: true
-  parent: ClothingBackpackERTEngineer
-  id: ClothingBackpackERTEngineerFilled
-  components:
-    - type: StorageFill
-      contents:
-        - id: BoxSurvival
-        - id: trayScanner
-        - id: WeaponPulseCarbine
-        - id: MedicatedSuture
-        - id: RegenerativeMesh
-        - id: RCD
-        - id: RCDAmmo
-        - id: RCDAmmo
-        - id: RCDAmmo
-        - id: CableMVStack
-        - id: CableHVStack
-
-- type: entity
-  noSpawn: true
-  parent: ClothingBackpackERTJanitor
-  id: ClothingBackpackERTJanitorFilled
-  components:
-    - type: StorageFill
-      contents:
-        - id: BoxSurvival
-        - id: WeaponPulsePistol
-        - id: MedkitFilled
-        - id: BoxLightMixed
-        - id: BoxLightMixed
-        - id: Soap
-        - id: CrowbarRed
-        - id: AdvMopItem
-
-- type: entity
-  noSpawn: true
-  parent: ClothingBackpackSalvage
-  id: ClothingBackpackSalvageFilled
-  components:
-    - type: StorageFill
-      contents:
-        - id: BoxSurvival
-
-- type: entity
-  parent: ClothingBackpackSatchelLeather
-  id: ClothingBackpackPirateFilled
-  suffix: Filled, Pirate
-  components:
-    - type: StorageFill
-      contents:
-        - id: BoxSurvival
-        - id: Cutlass
-        - id: WeaponRevolverPirate
-        - id: ClothingEyesEyepatch
+- type: entity
+  parent: ClothingBackpack
+  id: ClothingBackpackFilled
+  noSpawn: true
+  components:
+  - type: StorageFill
+    contents:
+      - id: BoxSurvival
+
+- type: entity
+  parent: ClothingBackpackAtmos
+  id: ClothingBackpackAtmosFilled
+  noSpawn: true
+  components:
+  - type: StorageFill
+    contents:
+      - id: BoxSurvivalEngineering

```

Comment:
bruh. just put it at the and of file, so diff would be not so big, and same in duffelbag.yml and satchel.yml